net/mini_snmpd/Makefile

```diff
@@ -8,46 +8,58 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mini_snmpd
-PKG_VERSION:=1.2b
-PKG_RELEASE:=8
+PKG_VERSION:=1.4-rc1
+PKG_RELEASE:=3
+PKG_MAINTAINER:=Luke McKee <hojuruku@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
 
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/troglobit/mini-snmpd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=203d92e60ed09466d6676c6ad20ad6cb2ce08a5d
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://members.aon.at/linuxfreak/linux/
-PKG_MD5SUM:=9e432c50ba8216d7fab0983b11b7112a
 
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
+PKG_FIXUP:=autoreconf
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
+# As warned by upstream maintainer and gcc
+PKG_SSP:=0
+PKG_RELRO:=0
```

Comment:
gcc gave warnings about RELRO that's for sure. You can try it without and see for yourself.
http://troglobit.github.io/mini-snmpd.html
See building small binaries section. 
@troglobit disables SSP, but fortify source is all good.
Maybe he can comment. 

The upstream maintainer also says he doesn't like "-Os" but I didn't honor him there ;)

With this release comes added security, I can make the default behavior of mini_snmpd to bind to the uci "lan" interface if you want.